src/sharing/types.ts

```diff
@@ -25,8 +26,8 @@ export type ShareUpdate = Partial<
 
 export interface MetaObject {
     meta: {
```

Comment:
Would you make `meta` itself optional? now you're forced to send at least an empty object.

Reply:
Yes looks good to me